static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -42,8 +41,9 @@ import {formatTo, ProfilingFormatterUnit} from 'sentry/utils/profiling/units/uni
 import {useDevicePixelRatio} from 'sentry/utils/useDevicePixelRatio';
 import {useMemoWithPrevious} from 'sentry/utils/useMemoWithPrevious';
 
-import {FlamegraphWarnings} from './FlamegraphWarnings';
-import {ProfilingFlamechartLayout} from './profilingFlamechartLayout';
+import {FlamegraphDrawer} from './flamegraphDrawer/flamegraphDrawer';
+import {FlamegraphWarnings} from './flamegraphOverlays/FlamegraphWarnings';
+import {FlamegraphLayout} from './flamegraphLayout';
```

Comment:
Any reason we're going back to the flamegraph terminology? This mix of flamegraph vs flamechart terminology can be confusing for people unfamiliar with it.

Reply:
Yeah, lets stick to graph as it sounds like that is where the future direction might go. I also dont think there is much preventing us today from actually rendering a graph if we wanted to